modal task/script.js

```diff
@@ -0,0 +1,6 @@
+function toggle(){
+    let blur = document.getElementById("blur");
+    blur.classList.toggle('active')
+    let modal = document.getElementById("modal");
```

Comment:
lets use const instead of let here and lets go through the use of const and let

https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#:~:text=%60const%60%20is%20a%20signal%20that,always%20the%20entire%20containing%20function.